NineChronicles.Headless.Executable/Program.cs

```diff
@@ -476,11 +476,6 @@ IActionLoader MakeSingleActionLoader()
                                 })
                         );
 
-                    // worker
-                    if (arenaParticipantsSync)
```

Comment:
Could you please delete these unused arguments?

Reply:
They can be resolved in the next PR.

Reply:
We can create new GFI yeah

Reply:
@Atralupus I'll open an issue before the end of the day.

Reply:
@Atralupus I opened it as https://github.com/planetarium/NineChronicles.Headless/issues/2598